observatory-platform/observatory/platform/bigquery.py

```diff
@@ -937,7 +951,10 @@ def bq_select_columns(
     """
 
     assert_table_id(table_id)
-    project_id, dataset_id, table_id, _, _ = bq_table_id_parts(table_id)
+    project_id, dataset_id, table_id, shard_date = bq_table_id_parts(table_id)
+    if shard_date is not None:
+        table_id = f"{table_id}{shard_date.format('YYYYMMDD')}"
+
```

Comment:
replace table_id in this fn with table_name where appropriate (ie. where not referring to the FQ table ID)

Reply:
I don't know, I've come around to just calling it table_id, because BigQuery calls the fully qualified name a table_id and also just the specific identifier for the table itself.